monorepo-migration/convert-to-hg.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/sh
+
+set -e
+set -x
+
+CINNABAR=https://github.com/jcristau/git-cinnabar
+CINNABAR_BRANCH=detect-more-renames
+SOURCE_REPO=https://github.com/mozilla-mobile/firefox-android
+SOURCE_REPO_NAME=${SOURCE_REPO##*/}
+SOURCE_BRANCH=main
+DEST_REPO="$(readlink -f "$1")"
+DEST_SUBDIR=mobile/android
+EXPRESSIONS_FILE_PATH="$(dirname $(readlink -f "$0"))/data/message-expressions.txt"
+
+TEMPDIR=$(mktemp -d)
+#trap 'rm -rf ${TEMPDIR} EXIT'
+CINNABAR_PATH="${TEMPDIR}/git-cinnabar"
+SOURCE_REPO_PATH="${TEMPDIR}/${SOURCE_REPO_NAME}"
+
+setup_temporary_repo() {
+  git clone "${SOURCE_REPO}" "${SOURCE_REPO_PATH}"
+  git -C "${SOURCE_REPO_PATH}" config user.email=release@mozilla.com --config user.name="Mozilla Release Engineering"
```

Comment:
Nit: missing `--` to `config user.email`?

Reply:
Hmm, no, but this does look wrong, will fix.